src/ext/java/org/opentripplanner/ext/transmodelapi/mapping/PreferencesMapper.java

```diff
@@ -24,6 +31,24 @@ static void mapPreferences(
         setStreetReluctance(preferences, streetReluctance);
       }
     );
+
+    if (GqlUtil.hasArgument(environment, "maxAccessEgressDurationForMode")) {
+      preferences.withStreet(street -> {
+        var maxAccessEgressDuration = StreetPreferences.DEFAULT.maxAccessEgressDuration().copyOf();
```

Comment:
We should not use `StreetPreferences.DEFAULT` here, it can be overridden in config. Instead we should just pass in the values from the user into `withMaxAccessEgressDuration`. Perhaps we can modify it, so that we call the function once per mode in the array.